install-dev/upgrade/php/ps_1761_update_currencies.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+use PrestaShop\PrestaShop\Adapter\SymfonyContainer;
+use PrestaShop\PrestaShop\Core\Localization\CLDR\LocaleRepository;
+
+function ps_1761_update_currencies()
+{
+    // Force cache reset of languages (load locale column)
+    ObjectModel::disableCache();
+
+    /** @var Currency[] $currencies */
+    $currencies = Currency::getCurrencies(true, false);
+    $context = Context::getContext();
+    $container = isset($context->controller) ? $context->controller->getContainer() : null;
```

Comment:
I don't think there's a controller during upgrade, is it?

Reply:
Depending if you're using CLI or not.

Reply:
I took the code from `ps_1760_copy_data_from_currency_to_currency_lang`. I obviously can force the use SymfonyContainer though.

Reply:
> Depending if you're using CLI or not

Oh, that makes sense I guess